types/src/eth.rs

```diff
@@ -205,6 +205,10 @@ pub struct AccountProofWrapper {
     pub balance: Option<U256>,
     #[serde(rename = "codeHash")]
     pub code_hash: Option<H256>,
```

Comment:
fyi in https://github.com/scroll-tech/go-ethereum/pull/188 I renamed this to `keccakCodeHash` to avoid confusion. We can use either name in the JSON representation.

Reply:
Will update accordingly. Thanks